File: Cloud5S_WEB/src/app/@filter/Common/account.filter.ts
import {EROLE_CODES} from 'src/app/utils/constant/index';

export class AccountFilter {
  currentPage: number = 1;
  pageSize: number = 20;
  keyWord: string = '';
  companyCode: string | null = null;
  isActive: boolean | string = '';
}

export interface optionsGroup {
  id: string;
  name: string;
}

export class AccountGetAllFilter {
  GroupId: string = '';
  RoleCode: string = EROLE_CODES['LAI_XE'];
  KeyWord: string = '';
}

File: Cloud5S_API/DMS.Business/Filter/AD/AccountFilter.cs
using DMS.BUSINESS.Filter.Common;

namespace DMS.BUSINESS.Filter.AD
{
    public class AccountFilter : BaseFilter
    {
        public Guid? GroupId { get; set; }
        public string RoleCode { get; set; }
    }

    public class AccountFilterLite
    {
        public Guid? GroupId { get; set; }
        public string RoleCode { get; set; }
        public string KeyWord { get; set; }
        public bool? IsActive { get; set; }
    }
}
